atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/OracleDbTableFactory.java

```diff
@@ -44,11 +45,27 @@ public DbMetadataTable createMetadata(String tableName, ConnectionSupplier conns
     }
 
     @Override
-    public DbDdlTable createDdl(String tableName, ConnectionSupplier conns) {
+    public DbDdlTable createDdl(TableReference tableName, ConnectionSupplier conns) {
         return new OracleDdlTable(tableName, conns, config);
     }
 
     @Override
+    public DbTableInitializer createInitializer(ConnectionSupplier conns) {
+        /* No initialization required for Oracle */
+        return new DbTableInitializer() {
+            @Override
+            public void createUtilityTables() {
+                //no op
+            }
+
+            @Override
+            public void createMetadataTable(String metadataTableName) {
+                //no op - will be required for Oracle to work
```

Comment:
We currently don't that will be included in another PR when the testing of Oracle is turned on.  At the moment we are not going to allow OSS to turn on Oracle.  For the verification check, do you mean beyond creating them if they don't exist as we do for Postgres or are you recommending an additional check.